Gemfile.lock

```diff
@@ -2,36 +2,36 @@ GEM
   remote: http://rubygems.org/
```

Comment:
Gemfile.lock probably changed when i bundle updated to get rid of an error.  https://github.com/AgileVentures/LocalSupport/issues/150